modules/tinymce/src/themes/silver/test/ts/module/StickyHeaderStep.ts

```diff
@@ -159,6 +160,64 @@ const testStickyHeader = (toolbarMode: ToolbarMode, toolbarLocation: ToolbarLoca
       editor.options.unset('toolbar_sticky_offset');
     });
   });
+
+  context('Test inline editor editor with toolbar_mode: ' + toolbarMode, () => {
+    const element: SugarElement<HTMLElement> = SugarElement.fromHtml(`<p id="content">${Strings.repeat('some content ', 1000)}</p>`);
+    const setupElement = () => {
+
+      Insert.append(SugarBody.body(), element);
+
+      return {
+        element,
+        teardown: () => {
+          Remove.remove(element);
+        }
+      };
+    };
+    const hook = TinyHooks.bddSetupFromElement<Editor>({
+      plugins: 'fullscreen',
+      base_url: '/project/tinymce/js/tinymce',
+      toolbar: 'align | fontsize | fontfamily | blocks | styles | insertfile | forecolor | backcolor ',
+      menubar: false,
+      inline: true
+    }, setupElement);
+
+    PageScroll.bddSetup(hook.editor, 5000);
+
+    before(async () => {
+      // Need to wait for a fraction for some reason on safari,
+      // otherwise the initial scrolling doesn't work
+      await Waiter.pWait(100);
```

Comment:
What happens without this. Can we make this a waiter that tries to scroll until it can then resets the scroll to position 0. If this takes more than 100 ms on some run then this would flake now.

Reply:
removed